docs/feature_unicode.md

```diff
@@ -29,7 +29,11 @@ const uint32_t PROGMEM unicode_map[] = {
 Make sure that the order for both matches.
 
 ## UCIS_ENABLE
+<<<<<<<<
 
+========
+
+>>>>>>>>
```

Comment:
conflict debris?

Reply:
Good catch!  

And yeah, not sure how that crept in!